setup.py

```diff
@@ -0,0 +1,37 @@
+import pathlib
+from setuptools import setup, find_packages
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
+
+# This call to setup() does all the work
+setup(
+    name="abuu",
+    version="1.0.0",
```

Comment:
```suggestion
    version=abuu.__version__,
```

And add `import abuu` above

Reply:
I've added the versioning in abuu/__init__.py just a few minutes ago, so you might have to pull to see it take effect.